picard/ui/options/plugins3.py

```diff
@@ -394,6 +394,10 @@ def _update_plugins(self, plugins):
         if not plugins:
             return
 
+        # Prevent multiple simultaneous updates
+        if hasattr(self, '_update_queue') and self._update_queue:
```

Comment:
Could be simplified to:

```
if getattr(self, '_update_queue', None):
```

But just a suggestion / personal preference, not blocking the merge